test/functional/page_objects/management/saved_objects_page.ts

```diff
@@ -193,53 +192,55 @@ export function SavedObjectsPageProvider({ getService, getPageObjects }: FtrProv
 
     async getElementsInTable() {
       const rows = await testSubjects.findAll('~savedObjectsTableRow');
```

Comment:
Also I don't think we need `await` here as well, do we? As Promise.all() function already waits for all the promises to resolve before returning,  it will not have to wait for the await keyword to resolve before continuing and that way might be bit more efficient.